routes/register.go

```diff
@@ -272,8 +273,12 @@ func Register(c Config) {
 				r.Get("/drafts-to-complete", handlers.DraftsToComplete).Name("drafts_to_complete")
 				// dashboard recent activity component
 				r.Get("/recent-activity", handlers.RecentActivity).Name("recent_activity")
-				// dashboard candidate records component
-				r.Get("/candidate-records", handlers.CandidateRecords).Name("candidate_records")
+				// all candidate records
```

Comment:
`r.Use(ctx.RequireUser)` -> this should be restricted to curators only

Reply:
you mean to say that `/candidate-records/*` should be restricted or all the existing controllers under `r.Use(ctx.RequireUser)`?

Reply:
restricted to user role "curator"